ruby/vampires.rb

```diff
@@ -0,0 +1,131 @@
+#Interview with the Vampire
+
+require 'date' #import date method
+
+def survey_session(times) #Method to process nultiple employees - Release 3
+	counter = 0
+	while counter < times # Loop to call survey_form according to times
```

Comment:
Watch out for using words that are already defined in Ruby, like `times`. Speaking of `times` - if you know exactly how many times you want to run a loop, the `.times` loop is a great way to set the break condition of a loop based on an integer. `while` and `until` loops are great, but `times` already has that break condition built into it.